### PR TITLE
Small infrastructure improvements

### PR DESCRIPTION
**Pull request by sebader**
- Increase AFD latency to 1s to cater for the very distributed deployment of INT (brazil and sweden right now)
- Show how to enable DDoS protection which is recommended to use in production

### CODE REVIEW

File: src/infra/workload/releaseunit/modules/stamp/network.tf [hunk 1]
```diff
@@ -24,6 +24,13 @@ resource "azurerm_virtual_network" "stamp" {
   location            = azurerm_resource_group.stamp.location
   address_space       = [module.subnet_addrs.base_cidr_block]
 
+  # For production workloads we recommend to enable DDoS protection Standard plan when exposing your AKS ingress on public IPs
+  # You can link this to an existing DDoS protection plan or create a new one
+  # ddos_protection_plan {
+  #   id     = var.ddos_protection_plan_id
```
Comment (msimecek) [inline src/infra/workload/releaseunit/modules/stamp/network.tf:30]: Where is this variable declared and how does one get the value of the procection plan?

**Review by msimecek — CHANGES_REQUESTED** [review]

File: src/infra/workload/releaseunit/modules/stamp/network.tf (hunk 1 above)
Comment (sebader) [reply src/infra/workload/releaseunit/modules/stamp/network.tf:30]: damn, you dont miss a thing :D I was wondering if I should actually include all that, too? would be all commented out, too. do you want me to?

Comment (msimecek) [reply src/infra/workload/releaseunit/modules/stamp/network.tf:30]: Maybe just explain what needs to be done for this to work? "add this variable to variables.tf, get the plan ID from..."

Comment (msimecek) [reply src/infra/workload/releaseunit/modules/stamp/network.tf:32]: Better :) Now I've learned that it's supposed to be resource ID.

**Review by msimecek — APPROVED** [review]

**Pull request merged**